parkings/api/operator/parking.py

```diff
@@ -57,6 +57,10 @@ def update(self, instance, validated_data):
         return instance
 
     def validate(self, data):
+        if self.instance and (now() - self.instance.created_at) > settings.PARKINGS_TIME_EDITABLE:
+            if set(data.keys()) != {'time_end'}:
+                raise exceptions.PermissionDenied(_('Other than PATCHing "time_end" not allowed anymore.'))
```

Comment:
Is `PermissionDenied` correct to return here? Somehow I think `VaidationError` would make more sense and it would result in the correct 400 status code?

Also, for error string, maybe: `Grace period has passed. Only "time_end" can be updated via PATCH.` ?